SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -2432,7 +2435,7 @@ class CFVMFlowSolverBase : public CSolver {
    */
   inline su2double GetCSkinFriction(unsigned short val_marker, unsigned long val_vertex,
                                     unsigned short val_dim) const final {
-    return CSkinFriction[val_marker][val_dim][val_vertex];
+    return CSkinFriction[val_marker][val_vertex][val_dim];
```

Comment:
I like this change, always wondered why the switched order only for this variable.

Reply:
Yes, it looked weird. Maybe because of cache misses for 2D arrays that do not fit in L1 cache, but I think this is now mostly dealt with by the compiler optimization?  Not sure if this is still an issue on modern architecture.

Reply:
** may have a very different layout in memory than [][].
Compilers can do very little about dynamically allocated variables.

Reply:
true, so no problem to make this change.